### PR TITLE
feat: added landing page for Ecom app

### PR DESCRIPTION
**Pull request by ooreally**
feat: added landing page of Ecom app
Here is the ss
image

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
+:root{
+    --bg-primary:#aa4a2b;
+    --shadow-primary:#7a361f;
+    --white-color: white;
+    --smokewhite-color: whitesmoke;
```
Comment (sharmayatendra) [inline css/index.css:5]: use hex code here as well

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+*{
+    color:black;
+}
+body{
+    background-color: var(--white-color);
+}
+.flex{
+    display: flex;
+    /* margin-right: auto; */
```
Comment (sharmayatendra) [inline css/index.css]: remove commented code

File: css/index.css [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+}
+
+.gutter-vertical{
+    width: 2rem;
+}
+
+/* header styles */
+.nav-bar{
+    background-color:var(--bg-primary);
+    /* justify */
```
Comment (sharmayatendra) [inline css/index.css]: remove here as well

File: css/index.css [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+    padding-right: 2rem;
+}
+
+.nav-link{
+    color: whitesmoke;
```
Comment (sharmayatendra) [inline css/index.css]: don't hard colors

File: css/index.css [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+/* .home-img::before{
+    position: absolute;
+    content:"";
+    top:0;
+    left:0;
+    right:0;
+    bottom:0;
+    background-image:url("../assets/guitar6.jpg");
+    background-repeat:no-repeat;
+    background-size:cover;
```
Comment (sharmayatendra) [inline css/index.css]: remove

**Review by sharmayatendra — APPROVED** [review]
you can look at the suggested changes. good work

**Review by vikrantkode — COMMENTED** [review]
All looks good to me. Ready to merge

**Pull request merged**